switches/http/do_valgrind_macosx.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash -x
```

Comment:
This file can certainly be renamed. I think we can remove it, though, it has a hidden dependency on siege.